### PR TITLE
Set keep_alive_timeout as 10 seconds in ClickHouse testing server

### LINKED ISSUE #9300: Flaky test setup TestClickHouseConnectorTest.init
https://github.com/trinodb/trino/runs/3650369157

```
Error:  io.trino.plugin.clickhouse.TestClickHouseConnectorTest.init  Time elapsed: 32.713 s  <<< FAILURE!
java.lang.RuntimeException: Failed to insert data: ClickHouse exception, code: 1002, host: localhost, port: 49159; localhost:49159 failed to respond
	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:119)
	at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:476)
	at io.trino.testing.QueryAssertions.copyTable(QueryAssertions.java:339)
	at io.trino.testing.QueryAssertions.copyTable(QueryAssertions.java:331)
	at io.trino.testing.QueryAssertions.copyTpchTables(QueryAssertions.java:323)
	at io.trino.plugin.clickhouse.ClickHouseQueryRunner.createClickHouseQueryRunner(ClickHouseQueryRunner.java:68)
	at io.trino.plugin.clickhouse.TestClickHouseConnectorTest.createQueryRunner(TestClickHouseConnectorTest.java:50)
	at io.trino.testing.AbstractTestQueryFramework.init(AbstractTestQueryFramework.java:93)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
	Suppressed: java.lang.Exception: SQL: CREATE TABLE IF NOT EXISTS customer AS SELECT * FROM tpch.tiny.customer
		at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:479)
		... 19 more
Caused by: io.trino.spi.TrinoException: Failed to insert data: ClickHouse exception, code: 1002, host: localhost, port: 49159; localhost:49159 failed to respond
	at io.trino.plugin.jdbc.JdbcPageSink.finish(JdbcPageSink.java:193)
	at io.trino.operator.TableWriterOperator.finish(TableWriterOperator.java:214)
	at io.trino.operator.Driver.processInternal(Driver.java:405)
	at io.trino.operator.Driver.lambda$processFor$9(Driver.java:291)
	at io.trino.operator.Driver.tryWithLock(Driver.java:683)
	at io.trino.operator.Driver.processFor(Driver.java:284)
	at io.trino.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:1076)
	at io.trino.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:163)
	at io.trino.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:484)
	at io.trino.$gen.Trino_testversion____20210920_102607_3.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: ru.yandex.clickhouse.except.ClickHouseUnknownException: ClickHouse exception, code: 1002, host: localhost, port: 49159; localhost:49159 failed to respond
	at ru.yandex.clickhouse.except.ClickHouseExceptionSpecifier.getException(ClickHouseExceptionSpecifier.java:91)
	at ru.yandex.clickhouse.except.ClickHouseExceptionSpecifier.specify(ClickHouseExceptionSpecifier.java:55)
	at ru.yandex.clickhouse.except.ClickHouseExceptionSpecifier.specify(ClickHouseExceptionSpecifier.java:24)
	at ru.yandex.clickhouse.ClickHouseStatementImpl.sendStream(ClickHouseStatementImpl.java:855)
	at ru.yandex.clickhouse.ClickHouseStatementImpl.sendStream(ClickHouseStatementImpl.java:824)
	at ru.yandex.clickhouse.ClickHouseStatementImpl.sendStream(ClickHouseStatementImpl.java:817)
	at ru.yandex.clickhouse.ClickHousePreparedStatementImpl.executeBatch(ClickHousePreparedStatementImpl.java:335)
	at ru.yandex.clickhouse.ClickHousePreparedStatementImpl.executeBatch(ClickHousePreparedStatementImpl.java:320)
	at io.trino.plugin.jdbc.JdbcPageSink.finish(JdbcPageSink.java:177)
	... 12 more
Caused by: org.apache.http.NoHttpResponseException: localhost:49159 failed to respond
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:143)
	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:261)
	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:165)
	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:167)
	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:272)
	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:124)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:271)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:107)
	at ru.yandex.clickhouse.ClickHouseStatementImpl.sendStream(ClickHouseStatementImpl.java:849)
	... 17 more
```

### PR DESCRIPTION
**Pull request by ebyhr**
Fixes #9300

300 stress tests succeeded.

### CODE REVIEW

File: plugin/trino-clickhouse/src/test/resources/config.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<yandex>
+    <keep_alive_timeout>10</keep_alive_timeout>
```
Comment (findepi) [inline plugin/trino-clickhouse/src/test/resources/config.xml:3]: Why? add code comment here & in cmt message too.

File: plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java [hunk 2]
```diff
@@ -33,6 +34,7 @@ public TestingClickHouseServer()
     {
         // Use 2nd stable version
         dockerContainer = (ClickHouseContainer) new ClickHouseContainer(CLICKHOUSE_IMAGE)
+                .withCopyFileToContainer(forClasspathResource("config.xml"), "/etc/clickhouse-server/config.d/config.xml")
```
Comment (findepi) [inline plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java:37]: Is there /etc/clickhouse-server/config.d/config.xml in the image?
or we're creating a new file? then, can we give it a more specific name?

**Review by findepi — APPROVED** [review]

File: plugin/trino-clickhouse/src/test/resources/config.xml (hunk 1 above)
Comment (ebyhr) [reply plugin/trino-clickhouse/src/test/resources/config.xml]: The purpose is fixing flaky server as https://github.com/ClickHouse/clickhouse-jdbc/issues/717.

File: plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java (hunk 2 above)
Comment (ebyhr) [reply plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java]: It's a new file. I renamed to `custom.yml`.

**Pull request merged**